### ISSUE TITLE
Getting `Invalid torrent_storage_dir path: /volume2/docker/qbittorrent/qBittorrent/BT_backup` when using docker

### ISSUE THREAD

**Issue by lynndixon** [L4GSP1KE/Upload-Assistant#307]
I am seeing `Invalid torrent_storage_dir path: /volume2/docker/qbittorrent/qBittorrent/BT_backup` whenever I attempt to run the uploader.  I have verified that the path ` /volume2/docker/qbittorrent/qBittorrent/BT_backup` is the EXACT path to my BT_Backup folder, and I am using the following to mount the volume to the container:  

`-v /volume2/docker/qbittorrent/qBittorrent/BT_backup:/BT_backup`

What am I missing here?

**Comment by madmax76** [comment]
It seems to be not an actual problem. For me it was using the wrong fork of this repo, this one does not support a few new trackers.

1) Merge your fork locally
2) Build a new container (fixing some errors related to [PEP 668])
3) Same warning, but successful result

**Comment by lynndixon** [comment]
I dont think this is the issue, as I pull a new version of the image every time I run the docker commands.  I also delete the image out form my local registry when I am done.
